web/urls.py

```diff
@@ -10,6 +10,7 @@
 
 urlpatterns = [
     path("djadmin/", admin.site.urls),
+    path("admin/people/", include("people_admin.urls")),
```

Comment:
just a note: I moved the URLs to /admin/people/ and admin/people/{state}/matcher for now